ufl/finiteelement/mixedelement.py

```diff
@@ -125,6 +123,9 @@ def symmetry(self):
             error("Size mismatch in symmetry algorithm.")
         return sm or EmptyDict
 
+    def sobolev_space(self):
+        return min(e.sobolev_space() for e in self._sub_elements)
+
```

Comment:
Is this the right thing? Suppose I have `CG1 x DG0`. I think this will say that the whole thing is in `H1`

Reply:
Yes, looks like max might be what we actually want here?

Reply:
I think that's a safe conservative option. It might disallow implicit restrictions in some cases where they would actually work, but that can be dealt with in a later pass if someone cares enough.